charts/mattermost-team-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 3.9.1
+version: 3.9.2
```

this is a "new feature" need to bump to 3.10.0
```suggestion
version: 3.10.0
```